{"name": "test_run[run_case0]", "status": "broken", "statusDetails": {"message": "NameError: name 'run' is not defined", "trace": "cmdopt = 'run'\n\n    @pytest.fixture(scope=\"session\")\n    def start_appium_desired(cmdopt):\n>   \topt = eval(cmdopt)\nE    NameError: name 'run' is not defined\n\ntestcase/conftest.py:22: NameError"}, "parameters": [{"name": "run_case", "value": "{'TC_ID': 'HomePage', 'CASE_ID': 'Case_001', '是否运行': 'y', '用例名称': '输入', '测试数据': 'send=appium', '期望结果': 'appium', '描述': '首页', '备注': '首页'}"}], "start": 1587407157120, "stop": 1587407157120, "uuid": "de4f69ea-6ffa-45a5-9f99-ecf3cc2a4c52", "historyId": "1b1458eb6c9578bbae1b78493d33e22c", "fullName": "testcase.TestKeywords.TestKeyword#test_run[run_case0]", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "TestKeywords"}, {"name": "subSuite", "value": "TestKeyword"}, {"name": "host", "value": "zhangxiangkuideMacBook-Pro.local"}, {"name": "thread", "value": "97689-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.TestKeywords"}]}